src/plugins/stripe.js

```diff
@@ -180,6 +182,19 @@ async function createStripePlugin (app, config) {
         }
       }
     },
+    onUpdate: {
+      [customerTable]: async ({ objects, newValues }, { query }) => {
+        // updates the stripe customer according with the changes on local customer
+        objects.forEach(async object => {
+          const toUpdate = filterObject(newValues, customerKeys)
+          // if the updated fields match the customerKeys
+          if (Object.keys(toUpdate).length > 0) {
+            const { [customerTable]: [customer] } = await query({ [customerTable]: { reservedId: object.reservedId, get: ['stripeId'] } }, { admin: true, readOnly: true })
+            customer && customer.stripeId && await stripe.customers.update(customer.stripeId, filterObject(newValues, customerKeys));
```

Comment:
The idea is correct. unfortunately, you need to store the data of the update in an object in the `local`variable, and really apply the update on Stripe in the onSuccess, because we need those changes to happen ONLY if the full request didn't fail. Otherwise, we might end up with the 2 database being out of sync.